### ISSUE TITLE
Training on logits rather than tokens?

### ISSUE THREAD

**Issue by SinanAkkoyun** [artidoro/qlora#271]
Hey, I would like to train a student model from my teacher model (knowledge distillation for specualtive decoding).
Commonly, the student model is being trained on the teachers logits (soft "labels") rather than tokens (hard "labels")

How can I do that with qLora? Thank you so much!
